src/views/states/list/components/InterfacesTypeSection.tsx

```diff
@@ -77,7 +78,13 @@ const InterfacesTypeSection: FC<InterfacesTypeSectionProps> = memo(
                 )}
               </Td>
               <Td>{iface?.['mac-address'] || '-'}</Td>
-              <Td>-</Td>
+              <Td>
+                <Checkbox
+                  id={`lldp-enabled-${interfaceType}-${iface.name}`}
+                  isDisabled
```

Comment:
What is the reason of adding checkbox that is gonna be always disabled?

Reply:
From the description of this PR, it seems you need "to show lldp column enabled or disabled". I can see many ways how to achieve this. But let me ask you: what the UX say about how to display this info in this specific list?

Reply:
Ux said that it's a checkbox

Reply:
So is it the UX decision to display checkbox that is always disabled?

Reply:
Yes. It's disabled because you cannot change the value. The state resource cannot be edited